src/main/java/com/google/maps/StaticMapApi.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.maps.errors.ApiException;
+import com.google.maps.internal.ApiConfig;
+import com.google.maps.internal.ApiResponse;
+import com.google.maps.model.StaticMapResult;
+
+import java.util.logging.Logger;
+
+/**
+ * Created by bhalchandrawadekar on 24/03/2017.
+ */
+
+public class StaticMapApi {
```

Comment:
I'm not seeing any tests for this new functionality. Can you please add some tests?